content/en/installing-python-and-numpy-guide.md

```diff
@@ -158,33 +158,33 @@ importing it in notebooks).
 
 ## NumPy packages & accelerated linear algebra libraries
 
-NumPy doesn't depend on any other Python packages, however it does depend on an
+NumPy doesn't depend on any other Python packages, however, it does depend on an
 accelerated linear algebra library - typically
 [Intel MKL](https://software.intel.com/en-us/mkl) or
-[OpenBLAS](https://www.openblas.net/).  The user doesn't have to worry about
+[OpenBLAS](https://www.openblas.net/). Users doesn't have to worry about
```

Comment:
"Users don't ..."

Reply:
I was focusing on the overall writing style of this page, and neglected the basics. This is what fatigue does to me.:( My apologies.

Reply:
no worries at all, that's what review is for:) 

> This is what fatigue does

same here